suites/quincy/rbd/tier-3_rbd_mirror_upgrade_scale.yaml

```diff
@@ -0,0 +1,193 @@
+      module: test_cephadm.py
+      name: deploy cluster
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph-rbd1:
+          config:
+            command: add
+            id: client.1
+            node: node2
+            install_packages:
+              - ceph-common
+              - rbd-nbd
+              - fio
+            copy_admin_keyring: true
+        ceph-rbd2:
+          config:
+            command: add
+            id: client.1
+            node: node2
+            install_packages:
+                - ceph-common
+                - rbd-nbd
+                - fio
+            copy_admin_keyring: true
+      desc: Configure the client system 1
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph-rbd1:
+          config:
+            verify_cluster_health: true
+            steps:
+              - config:
+                  command: apply
+                  service: rbd-mirror
+                  args:
+                    placement:
+                      label: rbd-mirror
+        ceph-rbd2:
+          config:
+            verify_cluster_health: true
+            steps:
+              - config:
+                  command: apply
+                  service: rbd-mirror
+                  args:
+                    placement:
+                      label: rbd-mirror
+      desc: RBD Mirror daemon deployment using cephadm
+      destroy-clster: false
+      module: test_cephadm.py
+      name: deploy rbd-mirror daemon
+
+  - test:
+      abort-on-fail: True
+      desc: Verify snapshot mirroring with upgrade
+      name: Test upgrade on snapshot mirrored clusters
+      module: test_rbd_snapshot_mirroring_with_upgrade.py
+      polarion-id: CEPH-83574895
+      clusters:
+        ceph-rbd1:
+          config:
+            rep-pool-only: true
+            rep_pool_config:
+              num_pools: 1
+              num_images: 1
+              size: 1G
+              mode: image # compulsory argument if mirroring needs to be setup
+              mirrormode: snapshot
+              snap_schedule_levels:
+                - image
+              snap_schedule_intervals: #one value for each level specified above
+                - 1m
+              io_size: 200M
+            command: start
+            service: upgrade
+            verify_cluster_health: true
+        ceph-rbd2:
+          config:
+            rep-pool-only: true
+            rep_pool_config:
+              num_pools: 1
+              num_images: 1
+              size: 1G
+              mode: image # compulsory argument if mirroring needs to be setup
+              mirrormode: snapshot
+              snap_schedule_levels:
+                - image
+              snap_schedule_intervals: #one value for each level specified above
+                - 1m
+              io_size: 200M
+            command: start
+            service: upgrade
```

Comment:
can we make a way `ceph  version` to upgrade through test config

Reply:
for that, we will have to upgrade client node, which is not included as part of any of the upgrade test suites, so that would be DMFG's work.